packages/target-decisioning-engine/src/timings.js

```diff
@@ -1,3 +1,9 @@
 /* eslint-disable import/prefer-default-export */
 
 export const TIMING_GET_OFFER = "get_offer";
+
+export const TIMING_ARTIFACT_DOWNLOADED_TOTAL = "artifactDownloaded_total";
+export const TIMING_ARTIFACT_DOWNLOADED_FETCH = "artifactDownloaded_fetch";
+export const TIMING_ARTIFACT_GET_INITIAL = "artifactGetInitial";
+export const TIMING_ARTIFACT_READ_JSON = "artifactDownloaded_read_JSON";
```

Comment:
for consistency with other names it should probably be renamed
```suggestion
export const TIMING_ARTIFACT_READ_JSON = "artifactDownloaded_readJson";
```